lib/bundler/cli/common.rb

```diff
@@ -98,5 +98,25 @@ def self.clean_after_install?
       clean &&= !Bundler.use_system_gems?
       clean
     end
+
+    def self.set_path(path, if_given = false)
+      if path.nil?
+        Bundler.settings.set_command_option :path, nil unless if_given
```

Comment:
```
if path.nil? && !if_given
  Bundler.settings.set_command_option :path, nil
  return
end
```

i think reads better

Reply:
but that behaves differently, we need to return even if `if_given` if true